src/styles/index.css

```diff
@@ -1,9 +1,226 @@
 @import url("https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.7.2/css/all.min.css");
+@import "decanter";
 
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@custom-variant aria-current (&[aria-current="true"]);
+@custom-variant aria-current-page (&[aria-current="page"]);
+@custom-variant data-intrail (&[data-intrail="true"]);
 
+@theme {
+  --breakpoint-sm: 640px;
+  --breakpoint-md: 768px;
+  --breakpoint-lg: 1024px;
+  --breakpoint-xl: 1280px;
+  --breakpoint-2xl: 1536px;
+  --breakpoint-3xl: 1600px;
+
+  /* Container sizes */
+  --container-8xl: 80rem;
+  --container-9xl: 90rem;
+  --container-10xl: 100rem;
+  --container-11xl: 110rem;
+  --container-12xl: 120rem;
+  --container-13xl: 130rem;
+  --container-14xl: 140rem;
+  --container-15xl: 150rem;
+
+  --font-sans: var(--font-source-sans);
+  --font-serif: var(--font-source-serif);
+  --font-stanford: var(--font-stanford);
+
+  --font-lato: var(--font-lato);
+  --font-merriweather: var(--font-merriweather);
+}
+
+@layer utilities {
```

Comment:
So after digging into this a bit,  and this doesn't work the way it did in v3 TW. Using layers locks in the styles and they can't be used with variants such as responsive classes. ie we wouldn't be able to do `lg:grid-cols-1-1`. The only way to make it work with variants like that is to use the `@utility class-name {}` syntax. I even tried changing the layer to `@layer utilities` but it still doesn't work.

https://tailwindcss.com/docs/upgrade-guide#adding-custom-utilities

I think we need to change many of the layers into `@utility`. This might also be need to be done in Decanter.